lib/uint.fz

```diff
@@ -110,7 +110,7 @@ is
   # returns the quotient and the remainder
   # NYI performance: https://cs.opensource.google/go/go/+/refs/tags/go1.19:src/math/big/natdiv.go
   fixed divide_with_remainder (divisor uint) tuple uint uint
-  pre divisor > uint.zero
+  pre safety: divisor > uint.zero
```

Comment:
here `safety` is probably justified, the code might run into an recursion or loop if we divide by zero.